app/bundles/PageBundle/EventListener/BuildJsSubscriber.php

```diff
@@ -382,12 +378,15 @@ public function onBuildJsForVideo(BuildJsEvent $event)
         }
        
         var cookieName = 'mautic-player-'+i+'-'+node.dataset.formId;
```

Comment:
I think we need to generate a random cookie name here for players without an associated form - what are your thoughts?